lblod/frontend-embeddable-notule-editor
# CSS: Convert rem to px values

## Overview
This changes the scss build step to Postcss with the inclusion of a Postcss plugin which converts all rem units to px units.

The reason for this change is to enable the development of a webcomponent.
In a webcomponent we have no access to the font-size of the html element. All rem units are based on the font-size of the html element which causes incorrect sizing of the interface.
Converting these rem units to px units fixes this issue.

### How to test/reproduce
Build should have no visual regressions. In the compiled styles all rems are now px.

This PR is necessary to fix the styling in https://github.com/lblod/poc-say-webcomponent/pull/1
